libs/domains/variables/feature/src/lib/variables-action-toolbar/variables-action-toolbar.tsx

```diff
@@ -0,0 +1,67 @@
+import { ActionToolbar, DropdownMenu, Icon, Tooltip } from '@qovery/shared/ui'
+
+export interface VariablesActionToolbarProps {
+  onImportEnvFile: () => void
+  onCreateVariable: () => void
+  onCreateVariableFile: () => void
+}
+
+export function VariablesActionToolbar({
+  onImportEnvFile,
+  onCreateVariable,
+  onCreateVariableFile,
+}: VariablesActionToolbarProps) {
+  return (
+    <ActionToolbar.Root>
```

Comment:
By default I think we can add `flex` for `ActionToolbar.Root` otherwise we have this case 

<img width="304" alt="Capture d’écran 2024-03-06 à 13 31 38" src="https://github.com/Qovery/console/assets/533928/727ee5e3-cf94-4f58-8362-f02f4c37ebfe">

Reply:
I don't have the problem so I blindly trust you
https://github.com/Qovery/console/pull/1227/commits/20e8b257ca399589682b2136922530e819f500c0